src/components/NewFeatures/NewFeatures.js

```diff
@@ -1,27 +1,23 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
 
 const NewFeatures = () => {
   return (
     <>
       <Helmet>
         <meta name="robots" content="noindex,nofollow" />
       </Helmet>
-      <header className="hero-banner report-filters">
-        <div className="dp-container">
-          <div className="dp-rowflex">
-            <div className="col-sm-12 col-md-12 col-lg-12">
-              <h2>Funcionalidades Nuevas</h2>
-              <p>
-                Features sin acceso para los usuarios todavía y con trabajo pendiente de contenidos
-                y diseño.
-              </p>
-            </div>
-          </div>
-          <span className="arrow"></span>
+      <HeaderSection>
+        <div className="col-sm-12 col-md-12 col-lg-12">
```

Comment:
This div cannot be part of the HeaderSection?

Reply:
Because in some pages use two times, like in reports

Reply:
👍 
In the future, we could parametrize it, something like this:

```html
       <HeaderSection col-sm=12 col-md=12 col-lg=12>
```